packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -487,11 +539,40 @@ export function makeVirtualObjectManager(
       if (initializing) {
         cache.remember(innerSelf);
       }
-      const representative = Far(tag, actualize(wrappedData));
+      const self = actualize(wrappedData);
+      let representative;
+      let exposedRepresentative;
```

Comment:
`stored` and `returned` don't quite work because they are both returned here (the returning that the proposed name refers to happens a layer or two above in the call stack).  I tried it with `storedRepresentative` and `exposedRepresentative` and I like how that reads.  I think I'm going to go with that for now, at least until the next review pass.